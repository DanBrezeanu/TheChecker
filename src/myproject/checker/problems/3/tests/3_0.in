5
9245 9397 706 7921 4462 
