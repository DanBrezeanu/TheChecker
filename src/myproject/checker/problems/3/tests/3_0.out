706 4462 
9397 9245 7921 
